generics.md

```diff
@@ -576,7 +576,7 @@ LinkedList<Member> getMembers() {
 }
 ```
 
-We can also use a method-level type parameter to link the return type to a parameter type:
+A method-level type parameter can also be used to link the return type to a parameter type (i.e., not just argument types):
```

Comment:
You mean "(i.e., not just to link two or more parameter types)"? This addition seems to make the sentence more complex at very limited added value; I just said above "Here, we need to link the two parameter types."